src/usb_moded-config.c

```diff
@@ -1085,6 +1095,15 @@ char * config_get_android_product(void)
     {
         return ssu_name;
     }
+#else
+    /*Trying use /etc/hw-release to get NAME*/
+    GKeyFile *hwReleaseFile;
+    GError *error = NULL;
+
+    hwReleaseFile = g_key_file_new();
+    if (g_key_file_load_from_file(hwReleaseFile, "/etc/hw-release", G_KEY_FILE_NONE, &error)) {
+        return g_key_file_get_string(hwReleaseFile, NULL, "NAME", NULL);
+    }
```

Comment:
Using a (local) helper function might improve readability vs. verbatim code duplication. Especially if it turns out that g_key_file_xxx() can't be used for this purpose and a custom parser needs to be written.